tests/e2e-playwright/utils/selectors.js

```diff
@@ -0,0 +1,31 @@
+const selectors = {
+    settingsHeader: "form[action='options.php'] h2",
+    settingsHeading: "Log in with Google Settings",
+    buttonSaveSetting: "role=button[name='Save Changes'i]",
+    messageSaveSetting: "#setting-error-settings_updated",
+    clientIdInput: '#client-id',
+    clientIdValue: '278082871881-qp9srgor0iqvl1aq200ctf1sdb49bli1.apps.googleusercontent.com',
```

Comment:
@alvitazwar Add this client ID and secret in `.env` or repo secret. This should not be publicly visible.